light-system-programs/tests/merkle_tree_tests.ts

```diff
@@ -732,7 +733,8 @@ describe("Merkle Tree Tests", () => {
             rent: DEFAULT_PROGRAMS.rent,
             transactionMerkleTree: TRANSACTION_MERKLE_TREE_KEY,
           })
-          .remainingAccounts([leavesPdas[1]])
+          // @ts-ignore
+          .remainingAccounts(leavesPdas[1])
```

Comment:
I think the fix is to do:
```
.remainingAccounts([leavesPdas[1]])
``` 
can you try this and remove the ts ignore?

Reply:
fixed as you said ✅